src/styles/index.scss

```diff
@@ -1,3 +1,81 @@
+@import 'variables';
+
 body {
   margin: 0;
+  .block {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    .stopwatch {
+      width: $stopwatchDiam;
+      height: $stopwatchDiam;
+      border: 1vmin dotted $borderColor;
```

Comment:
The unit 'vmin' for the border width might not be the best choice for consistency across different devices and screens. Consider using a fixed unit like 'px' for borders to ensure they appear consistently.